recipe/meta.yaml

```diff
@@ -76,20 +76,24 @@ outputs:
         - pickle5  # [py<38]
       run:
         - python
+        - aiosignal
         - attrs
         - click >=7.0
         - colorama
         - dataclasses  # [py<37]
         - filelock
+        - frozenlist
         - grpcio >=1.28.1, <=1.43.0
         - msgpack-python >=1.0.0, <2.0.0
         - numpy >=1.20
         - pickle5  # [py<38]
         - protobuf >=3.15.3
         - psutil
         - pyyaml
-        - redis-py >=3.5.0
+        - redis-py >=3.5.0  # [win]
+        - requests
         - setproctitle =1.2.2
+        - virtualenv
```

Comment:
It seems the Ray runtime env requires this to work: https://docs.ray.io/en/latest/ray-core/handling-dependencies.html#runtime-environments
So it will actually be required on runtime, not just for install. It could be that this is at the wrong location upstream, I'll check

Reply:
Ah it seems like `install_requires` is the correct location - it's just the core dependencies. That's a bad choice of naming from setuptools but it's nothing we can change :-)

Reply:
It's interesting that this supports conda environments natively, but still wants virtualenv. I'm guessing that's for the case when `RuntimeEnv(pip=...)`